src/realm/object-store/property.hpp

```diff
@@ -96,6 +96,8 @@ struct Property {
     IsPrimary is_primary = false;
     IsIndexed is_indexed = false;
     IsFulltextIndexed is_fulltext_indexed = false;
+    using NestedTypes = std::vector<CollectionType>;
+    NestedTypes nested_types;
```

Comment:
`operator==(Property const& lft, Property const& rgt)` needs to be updated to check equality of this new member

Reply:
:+1: